cql/DisplayCervicalCancerMedicalHistory.cql

```diff
@@ -91,9 +92,7 @@ code "Systemic lupus erythematosus ICD-9-CM Code": '710.0' from "ICD-9" display
 code "Chronic GvHD ICD-9 Code": '279.52' from "ICD-9" display 'Chronic graft-versus-host disease' // AKA: Diagnosis of Chronic GvHD
 code "Chronic GvHD ICD-10 Code": 'D89.811' from "ICD-10" display 'Chronic graft-versus-host disease' // AKA: Diagnosis of Chronic GvHD
 code "Biopsy Report": '65753-6' from "LOINC" display 'Cervix Pathology biopsy report' // A.K.A. "Histology tests"
-code "Positive": '10828004' from "SNOMED-CT" display 'Positive (qualifier value)'
```

Comment:
The commit message says you're removing the two negative codes, but you in reality removed a negative and a positive one. Is this correct?

Reply:
"Negative HPV Finding" is deleted since it is a member of the new "HPV Negative Results" value set, but the more generic "Negative" code wasn't deleted since it's used [elsewhere](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/blob/l2-updates-february/cql/DisplayCervicalCancerMedicalHistory.cql#L670) in the context of defining a negative histology result. 

Deleting the generic "Positive" code was a leftover change that I probably should have made in [this commit](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/7/commits/14d4a4b8ca5b2f81474b938023fca6103d37160c), but essentially the "Positive" code was left over from a previous version of the CDS and wasn't actually being used anywhere. It previously was used to define a positive HPV test result, but that was replaced by a more specific ["Positive HPV Finding" code](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/7/commits/14d4a4b8ca5b2f81474b938023fca6103d37160c#diff-e868c63c7171b5f74a516c013aa866a1db406f8a8a1018f8125fb13f0e3940adL96), which eventually got lumped into the ["High Risk HPV Positive Results" value set](https://github.com/ccsm-cds-tools/ccsm-cds-with-tests/pull/7/commits/14d4a4b8ca5b2f81474b938023fca6103d37160c#diff-e868c63c7171b5f74a516c013aa866a1db406f8a8a1018f8125fb13f0e3940adR81).